phone-test/src/gql/client/helpers.ts

```diff
@@ -0,0 +1,51 @@
+import { SubscriptionClient } from 'subscriptions-transport-ws';
```

Comment:
this file contains helpers used within the gql client instantiation, in the index file. Made sense to separate these, to allow for later ease of reuse and code organization